spaceobjects/templates/spaceobjects/index.html

```diff
@@ -74,36 +94,47 @@ <h4 class="subheading">Compiled data &amp; simulations for {{object_count|intcom
 </div>
 <div class="row">
   <div class="item-container">
-    <div class="item-container__inner">
       <h2>Asteroids and Comets</h2>
-      <div>
-        <p>
-          We've organized hundreds of thousands celestial objects into these categories below for your perusal.
-        </p>
-      </div>
-      <div class="vis-container vis-container__home"></div>
-      <div class="vis-selector-container">
-        <div class="vis-selector col-md-4" data-showvis="LARGEST">
-          <h3>Largest</h3>
+      <p>
+        We've organized hundreds of thousands celestial objects into these categories below for your perusal.
+      </p>
+      <div class="sticky vis-container vis-container__home"></div>
+      {% for object_set in object_sets %}
+        <div class="vis-selector" data-showvis="{{object_set.name}}">
+          <h3>{{object_set.name}}</h3>
           <p>
-            These are among the largest and earliest discovered asteroids in our solar system.
+            {{object_set.description}}
           </p>
-          <p>
-            <ol>
-              <li><a href="/asteroid/1-ceres">1 Ceres</a></li>
-              <li><a href="/asteroid/4-vesta">4 Vesta</a></li>
-              <li><a href="/asteroid/2-pallas">2 Pallas</a></li>
-              <li><a href="/asteroid/10-hygiea">10 Hygiea</a></li>
-              <li><a href="/asteroid/704-interamnia-1910-ku">704 Interamnia</a></li>
-      <!--
-            {% for space_object in space_objects %}
-              <li><a href="/asteroid/{{space_object.slug}}">{{space_object.name}}</a></li>
+          <div class="carousel-flex-container">
+            <div class="carousel-flex-inner">
+            {% for object in object_set.data %}
+              <div class="tile carousel-item" data-slug="{{object.slug}}" onmouseover="if (typeof selectOrbit !== 'undefined') selectOrbit(this)"
+                onmouseout="if (typeof deselectOrbit !== 'undefined') deselectOrbit(this)">
+                <a href="/asteroid/{{object.slug}}">
+                  <div>
+                    <h5>{{object.name}}</h5>
+                    <div class="tile-content">
+                      <span class="label label-danger">{{object.get_size_adjective}}</span>
```

Comment:
This is now just `object.size_adjective`

Reply:
done